test_framework/scripts/longhorn-uninstall.sh

```diff
@@ -0,0 +1,69 @@
+
+  local PYTEST_COMMAND_ARGS='''"-s",
+                                 "--junitxml='${LONGHORN_JUNIT_REPORT_PATH}'",
+                                 "--include-uninstall-test",
+                                 "-k", "test_uninstall"
+                              '''
+
+  ## generate uninstall_test pod manifest
+  yq e 'select(.spec.containers[0] != null).spec.containers[0].args=['"${PYTEST_COMMAND_ARGS}"']' "${LONGHORN_TESTS_MANIFEST_FILE_PATH}" > ${LONGHORN_UNINSTALL_TESTS_MANIFEST_FILE_PATH}
+  yq e -i 'select(.spec.containers[0] != null).spec.containers[0].image="'${LONGHORN_TESTS_CUSTOM_IMAGE}'"' ${LONGHORN_UNINSTALL_TESTS_MANIFEST_FILE_PATH}
+  yq e -i 'select(.spec.containers[0] != null).metadata.name="'${LONGHORN_UNINSTALL_TEST_POD_NAME}'"' ${LONGHORN_UNINSTALL_TESTS_MANIFEST_FILE_PATH}
+  yq e -i 'select(.spec.containers[0].env != null).spec.containers[0].env += {"name": "LONGHORN_REPO_BRANCH", "value": "'${LONGHORN_REPO_BRANCH}'"}' "${LONGHORN_UNINSTALL_TESTS_MANIFEST_FILE_PATH}"
+  
+  kubectl apply -f ${LONGHORN_UNINSTALL_TESTS_MANIFEST_FILE_PATH}
+
+  # wait uninstall test pod to start running
+  while [[ -n "`kubectl get pod ${LONGHORN_UNINSTALL_TEST_POD_NAME} -o=jsonpath='{.status.containerStatuses[?(@.name=="longhorn-test")].state}' | grep -v \"running\|terminated\"`"  ]]; do
+    echo "waiting uninstall test pod to be in running state ... rechecking in 10s"
+    sleep 10s
+  done
+
+  # wait uninstall test to complete
+  while [[ -n "`kubectl get pod ${LONGHORN_UNINSTALL_TEST_POD_NAME} -o=jsonpath='{.status.containerStatuses[?(@.name=="longhorn-test")].state}' | grep \"running\"`"  ]]; do
+    kubectl logs ${LONGHORN_UNINSTALL_TEST_POD_NAME} -c longhorn-test -f --since=10s
+  done
+
+  # get uninstall test junit xml report
+  kubectl cp ${LONGHORN_UNINSTALL_TEST_POD_NAME}:${LONGHORN_JUNIT_REPORT_PATH} "${TF_VAR_tf_workspace}/${LONGHORN_UNINSTALL_TEST_POD_NAME}-junit-report.xml" -c longhorn-test-report
+}
+
+set_kubeconfig_envvar(){
+  ARCH=${1}
+  BASEDIR=${2}
+
+    if [[ ${ARCH} == "amd64" ]] ; then
+    if [[ ${TF_VAR_k8s_distro_name} == [rR][kK][eE] ]]; then
+      export KUBECONFIG="${BASEDIR}/kube_config_rke.yml"
+    elif [[ ${TF_VAR_k8s_distro_name} == [rR][kK][eE]2 ]]; then
+      export KUBECONFIG="${BASEDIR}/terraform/aws/${DISTRO}/rke2.yaml"
+    else
+      export KUBECONFIG="${BASEDIR}/terraform/aws/${DISTRO}/k3s.yaml"
+    fi
+  elif [[ ${ARCH} == "arm64"  ]]; then
+    export KUBECONFIG="${BASEDIR}/terraform/aws/${DISTRO}/k3s.yaml"
+  fi
+}
+
+main(){
+  set_kubeconfig_envvar ${TF_VAR_arch} ${TF_VAR_tf_workspace}
+  LONGHORN_UNINSTALL_TEST_POD_NAME="longhorn-test-uninstall" 
+  run_uninstall_longhorn_test
+
+  mv ${TF_VAR_tf_workspace}/longhorn-test-junit-report.xml ${TF_VAR_tf_workspace}/longhorn-test-junit-report-temp.xml
+  jrm ${TF_VAR_tf_workspace}/longhorn-test-junit-report.xml "${TF_VAR_tf_workspace}/*.xml"
```

Comment:
You can just follow the current way we have for aggregate the report by junit Jenkins plugin. However, I like you use the dependencies in the container image, but we have the pattern already, so I would suggest just following the existing one.

https://plugins.jenkins.io/junit/
```
junit `*.xml`
```

or check an example in test_framework/Jenkinsfile
```
summary = junit 'longhorn-test-upgrade-from-stable-junit-report.xml, longhorn-test-upgrade-from-transient-junit-report.xml, longhorn-test-junit-report.xml'
```